src/components/TaskCard/TaskCard.scss

```diff
@@ -30,4 +30,9 @@
         padding: 0.5rem;
         background-color: base.$bg-gray-50;
     }
+
+    &--complete &__title {
```

Comment:
Please follow the BEM structure where `--` should be used and where `__` should  be used

Reply:
Following the BEM architecture, I've defined `.task-card` as the Block here. `__title` is its Element, and the modifier here is `--complete`. So, to change the style of `__title` having the modifier `--complete` as its parent, I've chosen this approach `&--complete &__title {}`. So, it is going to be something like,

```css
<div className="task-card task-card--complete">
     <div className="task-card__title"></div>
     <div className="task-card__body"></div>
</div>

Reply:
I see the block and element both contains same code. But what's the use of color and text-decoration in block level? we can just add the css only in the elements (`__title` and `__body`) . It means the same